js/contact.js

```diff
@@ -160,28 +175,32 @@ function displayContact(contactRef) {
  * Clears all fields in the create modal
  */
 function clearCreate() {
-	const newModal = document.getElementById("new-form");
 
-	for (let i = 0; i < newModal.children.length; i++) {
-		const editChild = newModal.children[i];
+	// Remove validation
+	document.getElementById("new-form").classList.remove("was-validated");
 
-		// Skip over not standard views.
-		if (!("contactKey" in editChild.dataset)) {
-			continue;
-		}
+	// Create a functional parser object.
+	const parser = element =>
+		({
+			key: element.dataset.contactKey,
+			targetID: element.dataset.contactTarget
+		});
```

Comment:
This can probably just be a global constant since it's used in more than one function.

Reply:
True! Wasn't quite sure how the whole => thing worked, and if it could be globalized or not.

Reply:
“Arrow Functions” are basically a shorthand/lightweight version of a function.